style.css

```diff
@@ -1,3 +1,8 @@
+html {
+    width: 1000px;
```

Comment:
You are shooting in foot by limiting HTML width. Test your design on fullscreen window